src/status_im/contexts/wallet/collectible/view.cljs

```diff
@@ -157,12 +157,10 @@
             set-title-ref                  (rn/use-callback #(reset! title-ref %))
             animation-shared-element-id    (rf/sub [:animation-shared-element-id])
             collectible                    (rf/sub [:wallet/collectible-details])
-            collectible-owner              (rf/sub [:wallet/collectible-details-owner collectible])
+            collectible-owner              (rf/sub [:wallet/current-viewing-account])
```

Comment:
Are we sure the `current-viewing-account` always refelcts the colletible owner? for example, in the wallet home screen, the `current-viewing-account` may be `nil`

Reply:
Yes and this okay, subscription`:wallet/total-owned-collectible` takes care of this.